loan/roll-loan.md

```diff
@@ -0,0 +1,49 @@
+---
+order: 4
+label: "Roll Loan"
+---
+
+# Rolling a Loan
+
+### How to Roll a Loan
+
+To roll a loan, look for a roll proposal on your loan details page.
+
+### Why Roll a Loan?
+
+Rolling a loan can be a strategic financial decision for several reasons. Here's why you might need or want to roll a loan:
+
+=== Extended Loan Term
+
+- **Need More Time**: If you need more time to repay your loan, rolling it can extend the loan term, providing you with additional time to manage your finances without defaulting.
```

Comment:
would remove "defaulting" since this is not something that is part of the protocol